pkg/api/validation/validation_test.go

```diff
@@ -1215,7 +1270,21 @@ func TestValidateVolumes(t *testing.T) {
 			},
 		},
 		{
-			name: "valid ConfigMap with projection",
```

Comment:
@pmorie should I leave it like this or add another with "valid ConfigMap with projection" only, then?